packages/arb-validator/rollupvalidator/server.go

```diff
@@ -205,7 +205,22 @@ func (m *Server) CallMessage(ctx context.Context, args *validatorserver.CallMess
 	}
 
 	inbox := message.AddToPrev(value.NewEmptyTuple(), deliveredMsg)
-	assertion, steps := m.man.ExecuteCall(inbox, m.maxCallTime)
+
+	latestBlock := m.man.CurrentBlockId()
+	latestTime := big.NewInt(time.Now().Unix())
+	timeBounds := &protocol.TimeBounds{
+		LowerBoundBlock:     latestBlock.Height,
+		UpperBoundBlock:     latestBlock.Height,
+		LowerBoundTimestamp: latestTime,
+		UpperBoundTimestamp: latestTime,
+	}
+	assertion, steps := mach.ExecuteAssertion(
+		// Call execution is only limited by wall time, so use a massive max steps as an approximation to infinity
```

Comment:
Do we have to work about DoS attacks here, or should we be charging clients for this service somehow?

Reply:
As discussed synchronously, the issue raised here definitely exists, but it also exists in master as well and so should be handled separately from this PR